# Description: git commit to fix an escaping issue
# Ubuntu: https://launchpad.net/bugs/428642
# Upstream: http://bugzilla.gnome.org/show_bug.cgi?id=603731
--- gnome-control-center-2.30.1.orig/libslab/app-shell.c
+++ gnome-control-center-2.30.1/libslab/app-shell.c
@@ -750,14 +750,16 @@
 		gtk_container_add (GTK_CONTAINER (app_data->filtered_out_everything_widget), hbox);
 	}
 
-	str1 = g_strdup_printf ("<b>%s</b>", app_data->filter_string);
+	str1 = g_markup_printf_escaped ("<b>%s</b>", app_data->filter_string);
 	str2 = g_strdup_printf (_("Your filter \"%s\" does not match any items."), str1);
-	markup = g_markup_printf_escaped ("<span size=\"large\"><b>%s</b></span>\n\n%s",
+	markup = g_strdup_printf ("<span size=\"large\"><b>%s</b></span>\n\n%s",
 		_("No matches found."), str2);
 	gtk_label_set_text (app_data->filtered_out_everything_widget_label, markup);
 	gtk_label_set_use_markup (app_data->filtered_out_everything_widget_label, TRUE);
 	gtk_box_pack_start (GTK_BOX (containing_vbox), app_data->filtered_out_everything_widget,
 		TRUE, TRUE, 0);
+	g_free (str1);
+	g_free (str2);
 	g_free (markup);
 }
 

